configurator/backend/handlers/sso_auth.go

```diff
@@ -0,0 +1,42 @@
+package handlers
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/jitsucom/jitsu/configurator/authorization"
+	"net/http"
+)
+
+type SSOAuthHandler struct {
+	authService *authorization.Service
+}
+
+func NewSSOAuthHandler(authService *authorization.Service) *SSOAuthHandler {
+	return &SSOAuthHandler{
+		authService,
+	}
+}
+
+func (oh *SSOAuthHandler) Handler(c *gin.Context) {
+	code := c.Query("code")
```

Comment:
please move `code` empty validation to this place from oh.authService.SSOAuthenticate.